docs/release_notes.rst

```diff
@@ -35,12 +35,12 @@ runs on most Windows®, Linux® and macOS® platforms.
 `main Connector
 repository <https://github.com/rticommunity/rticonnextdds-connector>`__.
 
-Version 1.1.0
+Version 1.1.1
 ~~~~~~~~~~~~~
 
-*RTI Connector* 1.1.0 is built on `RTI Connext DDS 6.1.0 <https://community.rti.com/documentation/rti-connext-dds-610>`__.
+*RTI Connector* 1.1.1 is built on `RTI Connext DDS 6.1.0 <https://community.rti.com/documentation/rti-connext-dds-610>`__.
 
-What's New in 1.1.0
+What's New in 1.1.1
```

Comment:
I don't think we should make any changes in the release_notes.rst file. For reference this file generates the docs here: https://community.rti.com/static/documentation/connector/current/api/python/release_notes.html
Maybe we could add a section for 1.1.1 if necessary, I'm not sure if that is something we do or not. I'll check with Elaine but in the meantime revert changes to this file